tests/src/Functional/NewsPageTest.php

```diff
@@ -22,7 +24,12 @@ class NewsPageTest extends BrowserTestBase {
    *
    * @var string
    */
-  protected $profile = 'standard';
+  protected $profile = 'localgov';
```

Comment:
Is there a particular reason to test using the profile.
I'm not sure its aggreed yet (one for technical group) if we should rely on the profile being installed, That would mean manually setting blocks etc... So its up to you if you want to do it, or just add the profile as a dev dependency.

Reply:
The news media test is the only one that fails when using the standard install profile.

I couldn't get to the bottom of why, and saw that the localgov profile is used in subsites testing so went with that. I'd prefer to understand why that test fails with the standard profile.